### PR TITLE
Added Push to refesh method

### PR DESCRIPTION
**Pull request by tolmadima**

### CODE REVIEW

File: app/src/main/java/com/example/lastfm/MainActivity.java [hunk 1]
```diff
@@ -45,6 +48,12 @@ public void onClick(View view) {
                 }
             }
         });
+        mSwipeRefreshLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+//                retrofitRequest();
```
Comment (ashvidchenko) [inline app/src/main/java/com/example/lastfm/MainActivity.java]: Why is this line commented out?

File: app/src/main/java/com/example/lastfm/MainActivity.java [hunk 2]
```diff
@@ -24,12 +26,13 @@
     private static final String REQUEST_TYPE = "json";
     public static List<Artist> requestedArtists = new ArrayList<>();
 
-    ArtistAdapter artistsAdapter = new ArtistAdapter(this::onArtistClick);
```
Comment (ashvidchenko) [inline app/src/main/java/com/example/lastfm/MainActivity.java:26]: Why did u delete this param?

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        android:id="@+id/swiperefresh"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:id="@+id/artistsRecyclerView"
-    />
+        android:layout_height="wrap_content">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/artistsRecyclerView"
```
Comment (ashvidchenko) [inline app/src/main/res/layout/activity_main.xml]: rename to rv_artists

File: app/src/main/res/layout/list_item_view.xml [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:padding="@dimen/item_padding"
-    android:background="@color/background_colour"
-    >
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/list_item"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+    android:background="#ffffff" >
```
Comment (ashvidchenko) [inline app/src/main/res/layout/list_item_view.xml]: use a constant

File: app/src/main/res/layout/list_item_view.xml [hunk 5]
```diff
@@ -1,13 +1,12 @@
...
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="horizontal"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:padding="@dimen/item_padding"
-    android:background="@color/background_colour"
-    >
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/list_item"
+    android:layout_width="fill_parent"
```
Comment (ashvidchenko) [inline app/src/main/res/layout/list_item_view.xml]: use another value, fill_parent is deprecated

File: app/src/main/res/layout/list_item_view.xml (hunk 4 above)
Comment (tolmadima) [reply app/src/main/res/layout/list_item_view.xml]: Changed to constant

File: app/src/main/java/com/example/lastfm/MainActivity.java (hunk 1 above)
Comment (tolmadima) [reply app/src/main/java/com/example/lastfm/MainActivity.java]: ucommented

File: app/src/main/res/layout/list_item_view.xml (hunk 5 above)
Comment (tolmadima) [reply app/src/main/res/layout/list_item_view.xml]: Changed

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (tolmadima) [reply app/src/main/res/layout/activity_main.xml]: renamed

File: app/src/main/java/com/example/lastfm/MainActivity.java (hunk 2 above)
Comment (tolmadima) [reply app/src/main/java/com/example/lastfm/MainActivity.java:26]: Среда разработки предложила другой, более корректный, вариант записи данной строки кода

**Review by ashvidchenko — APPROVED** [review]

**Review by ashvidchenko — APPROVED** [review]

**Pull request merged**